src/features/search-bar/organisms/search-bar.tsx

```diff
@@ -6,6 +6,7 @@ import { useEvent } from 'effector-react/ssr';
 
 import * as model from '../models';
 import { Search } from '../molecules';
+import { avatarUri } from '../../../shared/constants';
 import { paths } from '../../../pages/paths';
```

Comment:
Некритично, но я бы уже начинал абсолютные пути юзать
```suggestion
import { avatarUri } from '@box/shared/constants';
import { paths } from '@box/pages/paths';
```